README.rst

```diff
@@ -239,6 +239,14 @@ default value:
 ``sticky_by_default = False``
   Determine whether pdb++ starts in sticky mode or not.
 
+``enable_tiny_display = False``
+  Determine whether pdb++ starts in tiny display mode or not.
+  It needs to ``sticky_by_default = True``
+
+``divinding_line = '-'``
+  The divinding line to use with tiny display mode.
```

Comment:
*dividing ?

Reply:
oh, I'm sorry. 
It's typo.
`dividing` is right!